integrations/salesforce/hub.md

```diff
@@ -0,0 +1,58 @@
+The Salesforce Leads integration allows you search, create, update, and delete Salesforce contacts in your bot
+
+[![image](https://i.imgur.com/v5l26vi.png)](https://youtu.be/fPORGBUJmG0?si=uZLh40GSFbmGpPZE)
+
+## Integration Setup
+
+1. Enable the integration in your settings.
```

Comment:
suggestion for the future : I find it easier to combine the two. Something like: "toggle on the integration for your bot, and click the save button"

Reply:
Implemented